frontend/package.json

```diff
@@ -111,6 +111,7 @@
     "@types/lodash-es": "4.17.x",
     "@types/node": "12.12.14",
     "@types/react": "^17.0.1",
+    "@types/react-datepicker": "^4.15.0",
     "@types/react-dom": "^17.0.1",
```

Comment:
Why react-date-picker?

Reply:
When I was adapting react-datepicker to dark theme I had this warning message from the editor:
```
Could not find a declaration file for module 'react-datepicker'. '/home/fhoyosle/kiali/kiali_sources/kiali/frontend/node_modules/react-datepicker/dist/index.js' implicitly has an 'any' type.
Try `npm i --save-dev @types/react-datepicker` if it exists or add a new declaration (.d.ts) file containing `declare module 'react-datepicker'; ts(7016)`
```
For this reason I have added the types declaration of react-datepicker library to package.json